modules/apps_api/app/workers/apps_api/directory_loader.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require 'sidekiq'
+require 'rake'
```

Comment:
Requiring rake shouldn't be required here I don't think?

Reply:
ah this is leftover code! whoops